specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/2016-06-01/vaults.json

```diff
@@ -511,25 +499,20 @@
       "type": "object",
       "properties": {
         "Name": {
-          "description": "Name of the operation.",
-          "type": "string",
-          "readOnly": true
+          "description": "Name",

```

Comment:
@sonathan it appears that this one is still showing up https://travis-ci.org/Azure/azure-rest-api-specs/jobs/262537716#L1457 therefore causing 1 more error in the Linter compared to the spec before this PR, you may want to update this one as well.